capstone/src/main/java/com/google/step/data/WeeklyPageView.java

```diff
@@ -0,0 +1,62 @@
+package com.google.step.data;
+
+/**
+ * A simple class that contains the information for a weekly page view.
+ */
+public class WeeklyPageView {
+
+    private final int week;
+    private final int year;
+    private final int count;
+
+    public WeeklyPageView(int week, int year, int count) {
+        this.week = week;
+        this.year = year;
+        this.count = count;
+    }
+
+    /**
+     * Equals method for comparisons
+     * @param other object to compare with
+     * @return boolean for true if they are equals and
+     *         false if they are not.
+     */
+    @Override
+    public boolean equals(Object other) {
```

Comment:
when you implement equals, always implement hashCode as well.

Reply:
Done for both classes.